.github/actions/upgrade-integration-test/action.yaml

```diff
@@ -0,0 +1,25 @@
+# Runs upgrade suite of integration tests
+# Requires as input IMG ENV variable for make target
+# Don't forget to use actions/checkout@v3 and actions/setup-go@v4 before
+
+name: 'Upgrade integration test'
+description: 'Runs upgrade suite of integration tests'
+inputs:
+  img-url:
+    description: 'Env variable required for make target'
```

Comment:
Might be also good to have a description what img-url is, you only added what it is used for. We can also think about adding an example to make it clear.